package.json

```diff
@@ -38,12 +38,13 @@
     "chalk": "^2.4.1",
     "commander": "^2.18.0",
     "ejs": "^2.6.1",
-    "express": "^4.16.3",
     "filesize": "^3.6.1",
     "gzip-size": "^5.0.0",
     "lodash": "^4.17.10",
     "mkdirp": "^0.5.1",
     "opener": "^1.5.1",
+    "polka": "^0.5.1",
```

Comment:
How does it reduce the total size of dependencies? I would like to see the numbers.

Reply:
https://bundlephobia.com/result?p=polka@0.5.2
VS
https://bundlephobia.com/result?p=express@4.17.1

If/when serve-static is replaced:

https://bundlephobia.com/result?p=sirv@1.0.1
VS
https://bundlephobia.com/result?p=serve-static@1.14.1